src/puppetlabs/ezbake/core.clj

```diff
@@ -427,7 +442,8 @@ Dependency tree:
                                                           "puppetlabs.trapperkeeper.main")
          :java-args                 (get-local-ezbake-var lein-project :java-args
                                                           "-Xmx192m")
-         :bootstrap-source          (get-local-ezbake-var lein-project :bootstrap-source :file)}))))
+                                    ; Convert to string so ruby doesn't barf on the hyphens
+         :bootstrap-source          (name (get-local-ezbake-var lein-project :bootstrap-source :bootstrap-cfg))}))))
```

Comment:
Hmm... yeah, that sucks.  But, I expect it to be a rare enough thing, and easy enough to fix, that it seems like code maintainability probably trumps those other concerns.  As long as it doesn't make the test *too* hard to write.